src/main.ts

```diff
@@ -69,13 +69,40 @@ app.config.globalProperties.$filters = {
     }
     return featureValue;
   },
+  groupFeatures(featureHierarchy: any) {
+    let features =[] as any;
+    let  featuresValue = '';
+    const featuresList = {} as any;
+    if (featureHierarchy) {
+      featureHierarchy.forEach((feature: any) => {
+        const featureSplit = feature ? feature.split('/') : [];
+        features.push(featureSplit[1]);
+        features = [...new Set(features)];
+        features.forEach((x: any)=>{
+          featureHierarchy.filter((featureItem: any) => featureItem.includes(featureSplit[0] +"/" +x+"/")).forEach((feature: any) => {
+            const featureSplit = feature ? feature.split('/') : [];
+            const featureValue = featureSplit[2] ? featureSplit[2] : '';
+            featuresValue += ", " + featureValue;
```

Comment:
Can we do it something like:
```suggestion
            featuresValue += featuresValue.length > 0 ? ", " + featureValue : featureValue;
```
and remove the slice logic.

Reply:
Improved the logic